edu.wpi.first.wpilib.plugins.java/resources/templates/build.xml

```diff
@@ -24,6 +24,7 @@
   <property file="${user.home}/wpilib/wpilib.properties"/>
   <property file="build.properties"/>
   <property file="${user.home}/wpilib/java/${version}/ant/build.properties"/>
+  <property file="${user.home}/wpilib/java/${version}/ant/ni_image.properties"/>
```

Comment:
Hmm, I see what you're saying @PeterJohnson. Yeah, this will break it. However, we could move this to the build.xml we ship, instead of the template build.xml, and then it wouldn't be a breaking change.